.flake8

```diff
@@ -19,5 +19,161 @@ extend-exclude =
     build,
     modules,
     .git
+    libraries/AP_Camera/examples/gst_rtsp_server.py
+    libraries/AP_Camera/examples/gst_rtsp_to_wx.py
+    libraries/AP_Camera/examples/gst_udp_to_rtsp.py
+    libraries/AP_Camera/examples/gst_udp_to_wx.py
+    libraries/AP_Declination/generate/generate.py
```

Comment:
I would prefer this markup in-file so someone editing the file knows straight-up it's not checked for flake8 correctness and might be tempted to fix it.

Reply:
`markup in-file` also seems better because `pre-commit run flake8 --all-files` seems to ignore `--extend-exclude=`.

Reply:
Done in:
* #30340